src/tests/class_tests/openms/source/TargetedSpectraExtractor_test.cpp

```diff
@@ -0,0 +1,837 @@
+it++->setIntensity(6705.41660838088f);
+
+it->setMZ(128.4);
+it++->setIntensity(1676.35415209522f);
+
+it->setMZ(128.88);
+it++->setIntensity(3352.70830419044f);
+
+it->setMZ(129);
+it++->setIntensity(3352.70830419044f);
+
+it->setMZ(129.12);
+it++->setIntensity(6705.41660838088f);
+
+it->setMZ(129.84);
+it++->setIntensity(5029.06245628566f);
+
+it->setMZ(129.96);
+it++->setIntensity(10058.1249125713f);
+
+it->setMZ(130.08);
+it++->setIntensity(31850.7288898092f);
+
+it->setMZ(130.2);
+it++->setIntensity(10058.1249125713f);
+
+it->setMZ(130.32);
+it++->setIntensity(1676.35415209522f);
+
+it->setMZ(130.44);
+it++->setIntensity(1676.35415209522f);
+
+it->setMZ(130.56);
+it++->setIntensity(3352.70830419044f);
+
+it->setMZ(132.12);
+it++->setIntensity(1676.35415209522f);
+
+it->setMZ(138);
+it++->setIntensity(1676.35415209522f);
+
+it->setMZ(139.08);
+it++->setIntensity(1676.35415209522f);
+
+it->setMZ(140.16);
+it++->setIntensity(3352.70830419044f);
+
+it->setMZ(144.12);
+it++->setIntensity(1676.35415209522f);
+
+it->setMZ(146.04);
+it++->setIntensity(3352.70830419044f);
+
+it->setMZ(146.16);
+it++->setIntensity(1676.35415209522f);
+
+it->setMZ(156);
+it++->setIntensity(1676.35415209522f);
+
+it->setMZ(156.12);
+it++->setIntensity(5029.06245628566f);
+
+it->setMZ(156.36);
+it++->setIntensity(1676.35415209522f);
+
+it->setMZ(173.76);
+it++->setIntensity(1676.35415209522f);
+
+it->setMZ(174);
+it++->setIntensity(1676.35415209522f);
+
+it->setMZ(174.12);
+it++->setIntensity(6705.41660838088f);
+
+it->setMZ(174.24);
+it++->setIntensity(11734.4790646665f);
+
+it->setMZ(174.36);
+it++->setIntensity(6705.41660838088f);
+
+it->setMZ(174.6);
+it++->setIntensity(1676.35415209522f);
+
+it->setMZ(175.08);
+it->setIntensity(1676.35415209522f);
+
+START_SECTION(getMZ())
+{
+  TEST_EQUAL(spectrum[0].getMZ(), 61.92)
+  TEST_EQUAL(spectrum[0].getIntensity(), 6705.41660838088f)
+  TEST_EQUAL(spectrum[1].getMZ(), 68.88)
+  TEST_EQUAL(spectrum[1].getIntensity(), 1676.35415209522f)
+  TEST_EQUAL(spectrum[6].getMZ(), 84.84)
+  TEST_EQUAL(spectrum[6].getIntensity(), 8381.7707604761f)
+  TEST_EQUAL(spectrum[71].getMZ(), 174.6)
+  TEST_EQUAL(spectrum[71].getIntensity(), 1676.35415209522f)
+  TEST_EQUAL(spectrum[72].getMZ(), 175.08)
+  TEST_EQUAL(spectrum[72].getIntensity(), 1676.35415209522f)
+}
+END_SECTION
+
+/////////////////////////////////////////////////////////////
+/////////////////////////////////////////////////////////////
+
+TargetedSpectraExtractor* ptr = 0;
+TargetedSpectraExtractor* null_ptr = 0;
+const String experiment_path = OPENMS_GET_TEST_DATA_PATH("TargetedSpectraExtractor_13C1_spectra0to100.mzML");
+const String target_list_path = OPENMS_GET_TEST_DATA_PATH("TargetedSpectraExtractor_13CFlux_TraML.csv");
+MzMLFile mzml;
+MSExperiment experiment;
+TransitionTSVReader tsv_reader;
+TargetedExperiment targeted_exp;
+mzml.load(experiment_path, experiment);
+tsv_reader.convertTSVToTargetedExperiment(target_list_path.c_str(), FileTypes::CSV, targeted_exp);
```

Comment:
I suggest to specify here `retentionTimeInterpretation` as either minutes or seconds